service/raft/group0_state_machine.cc

```diff
@@ -116,22 +117,25 @@ static bool should_flush_system_topology_after_applying(const mutation& mut, con
 }
 
 static future<> write_mutations_to_database(storage_proxy& proxy, gms::inet_address from, std::vector<canonical_mutation> cms) {
-    std::vector<mutation> mutations;
+    std::vector<std::pair<schema_ptr, frozen_mutation>> mutations;
```

Comment:
We have frozen_mutation_and_schema in db/view/view_update_generator.hh. Can extract it to frozen_mutation.hh.

Reply:
good idea, will do in next version.

Reply:
Hmm, in fact it was there all along since 9e14412528d